app/src/main/java/net/gsantner/markor/format/TextConverter.java

```diff
@@ -121,6 +122,16 @@ protected String putContentIntoTemplate(Context context, String content, boolean
         final String contentLower = content.toLowerCase();
         AppSettings appSettings = new AppSettings(context);
         boolean darkTheme = appSettings.isDarkThemeEnabled() && !isExportInLightMode;
+		
+		SimpleDateFormat format= new SimpleDateFormat("yyyy-MM-dd HH:mm");
+        String myDate = format.format(new Date()); //1969-12-31 16:00
+        String time = myDate.split(" ")[1];
+        String hour = time.split(":")[0];
+        int currentHour = Integer.parseInt(hour);
+        if(currentHour>=21 || currentHour<=6){
+            darkTheme = true;
+        }
```

Comment:
So the issue is that we have a light flash in view mode alone, and not in edit mode?

Either way, this will force darkTheme on those who don't want dark theme at all etc. Right?

Reply:
don't mind these PRs :smile: